tech/languages/python/multiple-pythons.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Multiple Pythons
+subsection: python
+order: 8
+---
+
+# Multiple Python Interpreters
+
+If you are working on a piece of Python software, you probably want to test it
+on multiple Python interpreters. On Fedora, that's easy: all you have to do is
+use `dnf` to install what you need. Currently Fedora has the following Pythons
+ready for you in the repositories:
+
+ * CPython 3.6
+ * CPython 3.5
+ * CPython 3.4
+ * CPython 3.3
+ * CPython 2.7
+ * CPython 2.6
+ * PyPy
+ * PyPy 3
+ * Jython
+ * MicroPython
+
+Quite a nest, isn't it?
+You can install them like this:
+
+```bash
+sudo dnf install python33  # to get CPython 3.3
+sudo dnf install python34  # to get CPython 3.4
+sudo dnf install python26  # to get CPython 2.6
+sudo dnf install pypy pypy3 jython python35  # to get more at once
+```
+
+After that, you can run an interactive console or your script with, let's say,
+CPython 3.4:
+
+```
+$ python3.4
+Python 3.4.3 (default, Jul 11 2016, 11:30:44) 
+[GCC 5.3.1 20160406 (Red Hat 5.3.1-6)] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> 
+```
+
+Note that most of the CPython versions, other than those provided by the
+`python2` and `python3` packages, are only provided for developer's convenience
+so you can run your test suites. Critical security bugs might not be always
+fixed and those interprets are not intended for production.
+
+## Getting it and running it all with tox
+
+[Tox](https://tox.readthedocs.io/) is tool that helps you test your Python code
+on multiple Pythons. If you install it on Fedora via the dnf package manager,
+you'll automatically get all the CPythons and PyPys:
+
+```bash
+sudo dnf install tox
+```
+
```

Comment:
What about here:

> This will only install the Python interpreters on Fedora 25 or newer, on older Fedoras, you need to install the packages specifically.

And later remove that note once F24 EoLs.